src/ui_interface.h

```diff
@@ -95,7 +95,7 @@ class CClientUIInterface
     boost::signals2::signal<void (CWallet* wallet)> LoadWallet;
 
     /** Show progress e.g. for verifychain */
-    boost::signals2::signal<void (const std::string &title, int nProgress)> ShowProgress;
+    boost::signals2::signal<void (const std::string &title, int nProgress, const std::function<void(void)>& cancel)> ShowProgress;
```

Comment:
use `nullptr` as default parameter for `cancel` to avoid adding `nullptr` to all non-callback `ShowProgress` calls?

Reply:
I couldn't set a default parameter in the signal, so I'm not sure it is possible to do that. Or I'm just doing something wrong.